Meziantou.Xunit.ParallelTestFramework.Tests/Meziantou.Xunit.ParallelTestFramework.Tests.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="../Meziantou.Xunit.ParallelTestFramework/Meziantou.Xunit.ParallelTestFramework.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+    <PackageReference Include="xunit" Version="2.4.2" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.5">
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>

```

Comment:
You can include the props file instead of redeclaring the same values

````
<Project Include="../Meziantou.Xunit.ParallelTestFramework/build/Meziantou.Xunit.ParallelTestFramework.props"
````

Reply:
I had to use a slightly different syntax, but good call!